KPI-team1-frontend/src/pages/LoginPage.tsx

```diff
@@ -31,7 +31,7 @@ export default function LoginPage(): JSX.Element {
             email: data.user.email,
           };
           setUser(userData); // Update the user state in the App component
-          navigate("/kpi/1"); // todo: will update this route after getting favorite circle (`/kpi/${circles[0]?.circle_user[0]?.circle_id}`)
+          navigate("/kpi/circles/1"); // todo: will update this route after getting favorite circle (`/kpi/${circles[0]?.circle_user[0]?.circle_id}`)
```

Comment:
The default circle is already available in user details - up to you to use it

Reply:
I'm having trouble getting this particular part to work as expected. Could you please review my code for this section and provide suggestions on how I can fix it? At the moment, it's not functioning properly.